image_provisioner/playbooks/build_ami.yaml

```diff
@@ -24,7 +24,7 @@
       volumes:
         - device_name: "{{ '/dev/sda1' if atomic | default(False) else '/dev/sdb' }}"
           volume_type: gp2
-          volume_size: 50
+          volume_size: 70
```

Comment:
The size of the docker device was previously a concern for qcow2.   We had to keep it down to 20GB or less else it impacted storage in OpenStack.   Is this no longer a concern?  Is 70GB really required?

Reply:
@jeremyeder ^

Reply:
@mffiedler what environment were you using where the size was a problem?  Deathstar had no CoW Ceph to back up the VMs.  Last test we used almost no space because of the CoW, even though each VM was technically 50GB in size (and we had 2k of them).

Reply:
For AWS, the EBS volumes are not COW.  gp2 is pretty cheap, but we're leaving behind a lot of snapshots currently by folks not understanding how AMIs work.  Not opposed to the size increase, just want to make sure we need it hard-coded in the playbook vs. in the inventory.

Reply:
@mffiedler I think it's a good idea to parametrize the volume_size, will update the PR.